src/picosvg/svg_reuse.py

```diff
@@ -129,6 +133,17 @@ def _affine_callback(affine, subpath_start, curr_pos, cmd, args, *_unused):
             new_y = 0
         args[x_coord_idx] = new_x
         args[y_coord_idx] = new_y
+
+        # Arc radii are, excitingly, NOT coords. However, the curvature is entirely different
+        # and nothing normalizes if they are not adjusted so try scaling rx/y proportionally to
+        # the change in magnitude of the respective basis vectors.
+        if cmd.upper() == "A":
+            x_basis = Vector(affine.a, affine.b)
+            y_basis = Vector(affine.c, affine.d)
```

Comment:
`picosvg.Affine2D` has `decompose_scale` method that splits an affine into its scale component and whatever remains, we use that in nanoemoji when we need to scale PaintRadialGradient radii. It does basically the same thing.

Reply:
I wonder if I subconsciously remembered that!